C++/test/algorithm/dynamic_programming/longest_subsequence.cpp

```diff
@@ -0,0 +1,15 @@
+#include "third_party/catch.hpp"
+#include "algorithms/dynamic_programming/longest_subsequence.hpp"
+
+using namespace std;
```

Comment:
Instead of `using namespace std`, reference the particular items, like `using std::cout`